pkg/lib/cockpit-components-table.jsx

```diff
@@ -242,7 +243,6 @@ ListingTable.defaultProps = {
     columns: [],
     rows: [],
     actions: [],
-    className: 'ct-table',
```

Comment:
That change deserves an explanation, and a separate commit. It's now hardcoded above in render(), but it's not clear what happens here. (Also, using `props` as a variable is confusing by itself, as one easily misreads it as `this.props`)

Reply:
Basically if you have it in the default props it would be overwritten when the user will pass another className prop. Thus we need it hardcoded.
I addressed the comment with the props naming in a separate commit, since it's like that before this PR unfortunately